src/pygeosx/PyGroup.hpp

```diff
@@ -26,6 +27,77 @@ namespace python
 /**
  *
  */
+
+
+static constexpr char const * PyGroup_getWrapperDocString =
+  "get_wrapper(self, path, default, /)\n"
+  "--\n\n"
+  "Return the `Wrapper` at the relative path ``path``; ``default`` is optional.\n\n"
+  "If no wrapper exists and ``default`` is not given, raise a ``ValueError``;\n"
+  "otherwise return ``default``.\n"
+  "\n"
+  "Parameters\n"
+  "__________\n"
+  "path : str\n"
+  "    The relative path of the wrapper to return.\n"
+  "\n"
+  "Returns\n"
+  "_______\n"
+  "Wrapper\n"
+  "    The wrapper at the relative path.";
+
+
+template< typename T >
+static PyObject * PyGroup_getWrapper( T * const self, PyObject * const args ) noexcept
+{
+  //VERIFY_NON_NULL_SELF( self );
```

Comment:
Replace these with the macro expansions.

Reply:
I did something using macro expansion but I don't know if this is what you had in mind

Reply:
I meant replace `VERIFY_NON_NULL_SELF(self)` with `PYTHON_ERROR_IF( self == nullptr, PyExc_RuntimeError, "Passed a nullptr as self.", nullptr )`, which is the macro expansion, and then include the necessary headers.

Reply:
Aaah ok that makes sense.